homework-13/index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Homework 13</title>
+</head>
+
+<body>
+    <div class="container">
+        <section class="section-1">
+            <input type=" text" class="input1" id="input"><button class="btn" id="btn">Start Timeout</button>

```

Comment:
All inputs should have an associated label, let's follow a11y conventions

```suggestion
            <input type="text" class="input1" id="input"><button class="btn" id="btn">Start Timeout</button>
```